rust/src/lib.rs

```diff
@@ -41,6 +41,11 @@ mod ffi {
         fn get_systemctl_wrapper() -> &'static [u8];
     }
 
+    // composepost.rs
+    extern "Rust" {
+        fn compose_postprocess_final(rootfs_dfd: i32) -> Result<()>;
+    }
```

Comment:
Aside: is there a better way to avoid having to add everything in the bridge explicitly and just have cxx.rs key off of the visibility of the functions?

Reply:
On the Rust side it's implemented as a proc macro so it only sees what's inside there - by design it doesn't scan your whole source code (unlike what we're doing with cbindgen, which spews a lot of errors about stuff it doesn't know about).

Avoiding two-definition thing would be nice but I assume the current design allows it to scale better to larger codebases.